client/pages/tracks.jsx

```diff
@@ -5,29 +5,47 @@ export default class Tracks extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      displayName: null,
-      timeRange: null,
-      clickedOn: null
+      timeRange: 'all-time'
     };
     this.handleClick = this.handleClick.bind(this);
   }
 
   handleClick(event) {
-    this.setState({
-      displayName: null,
-      timeRange: null,
-      clickedOn: true
-    });
+    if (event.target.id === 'six-months') {
+      const allTimeFilter = document.querySelector('#all-time');
+      const sixMonthsFilter = document.querySelector('#six-months');
+      allTimeFilter.className = 'time-range';
+      sixMonthsFilter.className = 'time-range green';
+      fetch('/api/tracks6months')
+        .then(res => res.json())
+        .then(userInfo => {
+          this.setState({
+            timeRange: '6 months',
+            track: userInfo
+          });
+        });
+    } else {
+      const allTimeFilter = document.querySelector('#all-time');
+      const sixMonthsFilter = document.querySelector('#six-months');
+      allTimeFilter.className = 'time-range green';
+      sixMonthsFilter.className = 'time-range';
+      fetch('/api/tracksalltime')
```

Comment:
Same, here. We don't want to reach around React and update the DOM, we let React do that by passing props (such as `className`) to the correct elements.